contracts/dk/DuelistStakingEarnBox.sol

```diff
@@ -0,0 +1,241 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.4 <0.9.0;
+pragma abicoder v2;
+
+import '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+import '@openzeppelin/contracts/token/ERC20/ERC20.sol';
+import '@openzeppelin/contracts/utils/Address.sol';
+
+contract StakingEarnBoxDKT {
+  using SafeERC20 for ERC20;
+  using Address for address;
+
+  /**
+   * numberOfLockDays is a number of days that
+   * user must be stack before unstacking without penalty
+   */
+  struct StakingCampaign {
+    uint64 startDate;
+    uint64 endDate;
+    uint128 returnRate;
+    uint256 maxAmountOfToken;
+    uint256 stakedAmountOfToken;
+    uint256 limitStakingAmountForUser;
+    address tokenAddress;
+    uint256 maxNumberOfBoxes;
```

Comment:
These two can be packed, to save space and gas.

Reply:
`limitStakingAmountForUser` is use for limit token staked per user.
`maxNumberOfBoxes` is used for limit boxes reward for specific campaign.
I think these variable can not be packed, can't we?